app/.meteor/packages

```diff
@@ -52,3 +52,4 @@ akryum:vue-blaze-template
 underscore
 vuejs:blaze-integration
 akryum:vue-sass
+littledata:synced-cron
```

Comment:
Just make sure this package is somewhat maintained, as a lot of Meteor specific packages are being abandoned. Where is the official repository for this project? I can't find a repo with the 1.5.1 release.

Reply:
@brylie Here is the repository: https://github.com/percolatestudio/meteor-synced-cron
It has MIT liscensing and over 70k installs showing on atmosphere and did not have a lot of open issues. Is there anything else I should look for?

Reply:
Well, just make sure there is a *recent* release, i.e. that it is not abandoned. It should be OK, but I have just had trouble with some parts of the Meteor ecosystem getting abandoned, like Blaze.

Reply:
Alright will take care of it.